app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala

```diff
@@ -56,6 +56,14 @@ class DefaultDmsSubmissionService @Inject() (
     extends DmsSubmissionService
     with Logging {
 
+  def getDmsMetaConfig[A: Configs](key: String): A =
+    configuration.underlying
+      .get[A](s"microservice.services.upscan-initiate.dms.$key")
+      .value
+
+  val queue: String        = getDmsMetaConfig[String]("classification-type")
```

Comment:
are these values actually in application.conf? I can see entries for the path `dms.queue-name` and `dms.business-area`, but not under `microserice.services.upscan-initial.dms`

Reply:
should `.get[A](s"microservice.services.upscan-initiate.dms.$key")` be `.get[A](s"dms.$key")` and should this `classification-type`  be `queue-name`?

Reply:
good spot! Yes they are but I changed the names actually. Let me fix that

Reply:
fixed + fixed other config typos